Add clarifications on how to run examples. (#215)
tauri-apps/examples: README.md

These examples are a great way to see how Tauri works with a variety of frameworks.

## How to run 

> How to run

Reviewer: ```suggestion
```

> ensure to read the readme of the example to check that your first-time setup is done.

Reviewer: ```suggestion
Ensure that you have read the readme of the example to check that your first-time setup is done.
```

The examples assume by default that the main Tauri repo is in the root of your drive ( on windows c:/tauri/)

> 

Reviewer: Maybe clarify that it does this for the Matrix build because this is the repo for our smoketests.

> To change a exsample to be self contained:

Reviewer: ```suggestion
To change an example to be self-contained:
```

> go to /example/src-tauri/cargo.toml

Reviewer: ```suggestion
Go to `/%example_path%/src-tauri/cargo.toml`:
```

and change the dependency 
```toml
tauri = { path = "../../../../../tauri", features = [ "edge" ] }
```
to
```toml

> tauri = { version = "0.5.1", features = [ "edge" ] }

Reviewer: should be "latest" otherwise we'll be maintaining this readme forever

Author: done
